### PR TITLE
Add order pages

### LINKED ISSUE #2176: Create "Orders" app to manage orders
The app should be able to:
- list all orders
- order details
- create a new order

### PR DESCRIPTION
**Pull request by dominik-zeglen**
Resolves #2176

Order list:
zrzut ekranu 2018-05-16 o 15 02 14
zrzut ekranu 2018-05-16 o 15 02 25

Order details:
zrzut ekranu 2018-05-16 o 15 02 34
screenshot-2018-5-16 storybook
screenshot-2018-5-16 storybook 1

### CODE REVIEW

File: saleor/static/dashboard-next/components/DateFormatter/DateFormatter.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+      );
+    }
+    if (showTooltip) {
+      return (
+        <Tooltip
+          title={momentDate.format(outputFormat || "DD.MM.YYYY HH:mm")}
+          placement="bottom"
+        >
+          <Typography className={classes.root}>
+            {momentDate.locale(locale || "en").fromNow()}
```
[the remark this answers is not in the document]
Comment (dominik-zeglen) [reply saleor/static/dashboard-next/components/DateFormatter/DateFormatter.tsx]: What if you were browsing in a language other than browser's default?

File: saleor/static/dashboard-next/orders/components/OrderHistory/OrderHistory.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+                    {event.params.shippingAddress.firstName}{" "}
+                    {event.params.shippingAddress.lastName}
+                    <br />
+                    {event.params.shippingAddress.companyName && (
+                      <>
+                        {event.params.shippingAddress.companyName}
+                        <br />
+                      </>
+                    )}
+                    {event.params.shippingAddress.streetAddress_1}
```
Comment (patrys) [inline saleor/static/dashboard-next/orders/components/OrderHistory/OrderHistory.tsx]: We should not format addresses this way. This assumes there is one universal address format which is not true. For now I'd count on the backend returning the address formatted correctly.

File: saleor/static/dashboard-next/components/DateFormatter/DateFormatter.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (patrys) [reply saleor/static/dashboard-next/components/DateFormatter/DateFormatter.tsx]: I'm talking about the `|| "en"` part.

[the remark this answers is not in the document]
Comment (dominik-zeglen) [reply saleor/static/dashboard-next/components/DateFormatter/DateFormatter.tsx]: 👍  point for you

File: saleor/static/dashboard-next/orders/components/OrderHistory/OrderHistory.tsx (hunk 2 above)
Comment (dominik-zeglen) [reply saleor/static/dashboard-next/orders/components/OrderHistory/OrderHistory.tsx]: Like formatted string?

Comment (patrys) [reply saleor/static/dashboard-next/orders/components/OrderHistory/OrderHistory.tsx]: Yes, until we are able to format addresses _properly_ we should not try to do it in the client.

**Pull request merged**